### PR TITLE
AEA-3338 add sandbox deployment

### PR DESCRIPTION
**Pull request by anthony-nhs**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,7 +50,10 @@ sam-list-outputs: guard-AWS_DEFAULT_PROFILE guard-stack_name
 sam-validate: 
 	sam validate
 
-sam-deploy-package: guard-artifact_bucket guard-artifact_bucket_prefix guard-stack_name guard-template_file guard-cloud_formation_execution_role guard-LATEST_TRUSTSTORE_VERSION guard-enable_mutual_tls
+sam-validate-sandbox: 
+	sam build --template-file sandbox_template.yaml
+
```
Comment (anthony-nhs) [inline Makefile:55]: this should be 
```
sam validate --template-file sandbox_template.yaml
```

File: Makefile [hunk 2]
```diff
@@ -56,20 +65,23 @@ sam-deploy-package: guard-artifact_bucket guard-artifact_bucket_prefix guard-sta
 		--no-fail-on-empty-changeset \
 		--role-arn $$cloud_formation_execution_role \
 		--no-confirm-changeset \
-		--force-upload \
-		--parameter-overrides ParameterKey=TruststoreVersion,ParameterValue=$$LATEST_TRUSTSTORE_VERSION ParameterKey=EnableMutualTLS,ParameterValue=$$enable_mutual_tls
+		--parameter-overrides TargetSpineServer=$$target_spine_server \
```
Comment (anthony-nhs) [inline Makefile:68]: this should be 
```
--parameter-overrides ParameterKey=TargetSpineServer,ParameterValue=$$target_spine_server  ParameterKey=TruststoreVersion,ParameterValue=$$LATEST_TRUSTSTORE_VERSION ParameterKey=EnableMutualTLS,ParameterValue=$$enable_mutual_tls
```

File: Makefile (hunk 1 above)
Comment (kris-szlapa) [reply Makefile:55]: The change has been implemented

File: Makefile (hunk 2 above)
Comment (kris-szlapa) [reply Makefile:68]: The change has been implemented

**Review by kris-szlapa — APPROVED** [review]

**Pull request merged**